tests/integration/debug_callback_view/test_views.py

```diff
@@ -0,0 +1,80 @@
+import json
+
+from django.http import HttpRequest
+
+from django_stomp.debug_callback_view.views import debug_callback_view
+from django_stomp.debug_callback_view.views import mock_ack
+from django_stomp.debug_callback_view.views import mock_nack
+from django_stomp.services.consumer import Payload
+
+fake_callback_path = "tests.unit.debug_callback_view.test_views.fake_callback"
+
+
+def fake_callback(payload: Payload):
+    return
+
+
+def test_should_return_error_when_method_not_is_post(mocker):
+    mock_fake_callback_function = mocker.patch(fake_callback_path)
+
+    mock_request = HttpRequest()
+    mock_request.method = "GET"
+
+    response = debug_callback_view(mock_request)
+
+    assert response.status_code == 200
+    assert response.content == b'{"status": "error", "detail": "Only POST method is allowed"}'
+
+    mock_fake_callback_function.assert_not_called()
+
+
+def test_should_return_error_when_callback_function_path_body_attribute_is_not_provided(mocker):
+    mock_fake_callback_function = mocker.patch(fake_callback_path)
+
+    mock_request = HttpRequest()
+    mock_request.method = "POST"
+    mock_request._body = json.dumps({})  # empty body
+
+    response = debug_callback_view(mock_request)
+
+    assert response.status_code == 200
+    assert response.content == b'{"status": "error", "detail": "callback_function_path is required"}'
+
+    mock_fake_callback_function.assert_not_called()
+
+
+def test_should_return_warning_when_callback_function_raise_error(mocker):
+    mock_fake_callback_function = mocker.patch(fake_callback_path)
```

Comment:
**suggestion**: Since this is being mocked on several tests, we could create a fixture to mock it.

```python
@pytest.fixture
def mock_fake_callback_function(mocker: MockFixture) -> MagicMock:
    return mocker.patch(fake_callback_path)
```

And the tests would be something like this:
```python
def test_xpto(mock_fake_callback_function: MagicMock):
    ...
    mock_fake_callback_function.assert_not_called()
```